bin/add_transform_test.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
+"""Automatically create transform tests input and output files given an input template.
+
+Usage:
+    add_transform_test.py --template-file=sam-template.yaml [--disable-api-configuration]
+    add_transform_test.py --template-file=sam-template.yaml
+
+Options:
+    --template-file=<i>             Location of SAM template to transform [default: template.yaml].
+    --disable-api-configuration     Disable adding REGIONAL configuration to AWS::ApiGateway::RestApi
+"""
+import json
+import subprocess
+import re
+import os
+import shutil
+import sys
+import yaml
+import tempfile
+from docopt import docopt
+from pathlib import Path
+from typing import Any, Dict
+
+from samtranslator.yaml_helper import yaml_parse
+
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+TRANSFORM_TEST_DIR = os.path.dirname(os.path.realpath(__file__)) + "/../tests/translator/"
```

Comment:
```suggestion
TRANSFORM_TEST_DIR = SCRIPT_DIR + "/../tests/translator"
```

The trailing slash would've made the script path `//` which I'm not sure is valid.